extensions.lua

```diff
@@ -152,6 +153,20 @@ local function process_config(conf)
         end
 
         for _, event in ipairs(fconf.events) do
+            if event.http == nil then
+                -- do nothing
```

Comment:
It should be covered with a test. `cartridge.cfg({http_enabled = false})` is an important corner case that usually results in a bug, see https://github.com/tarantool/migrations/issues/26.